src/Skybrud.Umbraco.Redirects/Config/RedirectsContentAppSettings.cs

```diff
@@ -13,6 +13,10 @@ public class RedirectsContentAppSettings {
         public bool Enabled { get; set; } = true;
 
         /// <summary>
+        /// Gets or sets whether the user's start nodes should filter which redirects they have access to. Default is <see lanword="true"/>.
+        /// </summary>
+        public bool UserStartNodes { get; set; } = false;
+
```

Comment:
Sorry - I made a mistake in my description about this property. My intention was to indicate whether to *use* starts nodes as the basis for permissions.

Could you rename this to `UseStartsNodes`?

The XML summary is also slightly wrong, as the default is `false` and not `true`.  And `lanword` should be `langword`.